source/includes/_endpoint-tabbook.md

```diff
@@ -1,24 +1,46 @@
 ## Tab books
 
-The `tabbook` view of a multitable will generate an excel (.xlsx) workbook containing each variable in the dataset crosstabbed with a given multitable. 
+The `tabbook` view of a multitable will generate an excel (.xlsx) workbook 
+containing each variable in the dataset crosstabbed with a given multitable. 
 
-A request to `datasets/{id}/multitables/{id}/tabbook/' will generate a download location to which the exporter will write this file when it is done computing (it may take some time for large datasets). Clients should note the download url, monitor progress, and when complete, GET the download location. 
+A request to `datasets/{id}/multitables/{id}/tabbook/' will generate a download 
```

Comment:
Also, here and throughout, please follow the URL conventions used elsewhere in the apidocs. (See https://github.com/Crunch-io/apidocs/blob/master/source/includes/_endpoint-dataset.md for example.) In examples, use a full URL, with the beta subdomain, like `https://beta.crunch.io/api/datasets/...`, and use ids instead of `{templates}`--they're examples. In the text, templates are fine, and omit the leading `https://beta.crunch.io/api` from URLs.